user_tools/src/spark_rapids_pytools/cloud_api/dataproc.py

```diff
@@ -107,13 +152,38 @@ def _list_inconsistent_configurations(self) -> list:
                     incorrect_envs.append(f'Property {prop_value} is not set.')
```

Comment:
small nit: should it be `incorrect_envs.append(f'Property {prop_entry} is not set.')`?